docs/data-guidelines.md

```diff
@@ -142,23 +142,31 @@ This guideline was proposed in [#6156](https://github.com/mdn/browser-compat-dat
 
 If you set `"partial_implementation": true`, then write a note describing how the implementation is incomplete.
 
+Typically, when the implementation is partial due to a bug or limitation in a browser, the note should describe that and any possible workaround, and preferably link to any open issue on the browser for them.
+
 This guideline was proposed in [#7332](https://github.com/mdn/browser-compat-data/pull/7332).
 
-## Non-functional defined names imply `"partial_implementation"`
+### Support status
+
+The decision as to whether something is unsupported, partially supported, or (fully) supported is ultimately an editorial decision.
+
+The simplest state to explain is “unsupported”: if a feature is entirely absent then it should be marked as unsupported  (`"version_added": false`).
+
+Distinguishing the other two is harder; but as some guidelines consider the following:
 
-If a browser recognizes an API name, but the API doesn’t have any discernable behavior, use `"partial_implementation": true` instead of `"version_added": false`, as if the feature has non-standard support, rather than no support.
+If a browser recognizes a feature, but the feature doesn’t have any discernable behavior, then it should be marked as partially supported (`"partial_implementation": true`), as if the feature has non-standard support, rather than no support.
 
-For example, suppose there is some specification for a Web API `NewFeature.method()`. Running `typeof NewFeature.method` in some browser returns `function` (not `undefined`), but the method, when called, returns `null` instead of an expected value. For that feature, set `"partial_implementation": true` and write a note describing the feature’s misbehavior.
+For example, suppose there is some specification for a Web API `NewFeature.method()`. Running `typeof NewFeature.method` in some browser returns `function` (not `undefined`), but the method, when called, returns `null` instead of an expected value. For that feature, set `"partial_implementation": true` and write a [note](#partial_implementation-requires-a-note) describing the feature’s misbehavior.
 
-See [#3904](https://github.com/mdn/browser-compat-data/pull/3904#issuecomment-484433603) for additional background (and nuance).
+If the implementation has bugs which it is likely a large percentage of web developers will encounter when attempting to use the feature, then it should be marked as a partial implementation with an appropriate [note](#partial_implementation-requires-a-note).
 
-## Operating system limitations imply `"partial_implementation"`
+If the implementation is only supported in specific contexts (for example, if `break-before` is only supported for printed pages but not for other types of fragmentation, such as multi-column layout), then it should be marked as a partial implementation with an appropriate [note](#partial_implementation-requires-a-note).
```

Comment:
This requires a bit more nuance. Sometimes the context itself didn't originally exist, such as service workers. We shouldn't mark things as partially supported when a new such context is added and existing features are defined to work in those contexts.

Reply:
Hmm, true; presumably, we want to define subfeatures when that happens ... but how do we differentiate that from stuff that new contexts get "for free", like (non-sharing-related) JavaScript builtins in new types of workers?